docs/development_controllers/chip-tool/chip_tool_guide.md

```
@@ -814,7 +815,8 @@ against a paired Matter device.
 See the [Examples](#running-testclusters-test) section for an example of how to
 run a test from the test suite.
 
-<hr>
+Developer details on how the test suite is structured can be found
+[here](https://github.com/project-chip/connectedhomeip/blob/master/src/app/tests/suites/README.md).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)